### ISSUE TITLE
[CLOSED] Privileges - refinement of the initial distribution set to be added

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#61]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Monday Jul 27, 2015 at 03:21 GMT_
_Originally opened as https://github.com/adamsmj/yawl/issues/61_

----

```
In the resource wizard of the yawl editor, the last dialog should ask for
each privilege whether the initial distribution set needs to be restricted
(hence 3 radio buttons should appear: "No", "Yes" and "Yes, refine"). If
"Yes, refine" is selected, a new dialog has to pop up showing the initial
distribution set and allowing the user to restrict it. This has to be done
for each privilege for which this option has been selected, leading to a
maximum of 12 dialogs for this wizard.
 
```

Original issue reported on code.google.com by `marcello...@gmail.com` on 31 Jul 2008 at 3:05

**Comment by yawlfoundation** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Monday Jul 27, 2015 at 03:21 GMT_

----



Original comment by `yawl.mic...@gmail.com` on 6 Aug 2008 at 7:26
* Added labels: **Category-Editor**

**Comment by yawlfoundation** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Monday Jul 27, 2015 at 03:21 GMT_

----

```
resolved in editor 3.0
```

Original comment by `yawl.mic...@gmail.com` on 9 Dec 2013 at 2:31
* Changed state: **Fixed**
